src/limestone-api.ts

```diff
@@ -130,18 +130,22 @@ export default class LimestoneApi {
    * {@link https://github.com/limestone-finance/limestone-api/blob/main/ALL_SUPPORTED_TOKENS.md}
    *
    * @param symbol - Token symbol
-   * @param opts - Options object. It must contain startDate, endDate, and interval properties.
+   * @param opts - Options object.
+   * It must contain either startDate, endDate, and interval properties
```

Comment:
It must contain either a startDate, an endDate, and interval properties
   * or an offset and a limit (used for pagination) properties.